src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtilsImpl.java

```diff
@@ -77,10 +77,10 @@ public void addInfo(TableRow row, VariantContext variantContext,
     // Iterate all Info field in VCFHeader, if current record does not have the field, skip it.
     for (VCFInfoHeaderLine infoHeaderLine : vcfHeader.getInfoHeaderLines()) {
       String attrName = infoHeaderLine.getID();
+      VCFInfoHeaderLine infoMetadata = vcfHeader.getInfoHeaderLine(attrName);
```

Comment:
nit: If the only use of `infoMetadata` is to call `getType()` then let's not create the object:
```
      VCFHeaderLineType infoType = vcfHeader.getInfoHeaderLine(attrName).getType();
```
Also let's move this line into the else statement, instead of extracting the type for all info fields we only do it when the attribute is missing. A little bit more efficient.

Reply:
InfoMetadata  will also be used to `getCountType()`(Integer, A, G, R, etc).

Reply:
I understand, thanks for clearing up.